lib/class-wp-block-supports.php

```diff
@@ -207,14 +207,25 @@ function get_block_wrapper_attributes( $extra_attributes = array() ) {
  * @return array Block attributes.
  */
 function wp_block_supports_track_block_to_render( $args ) {
-	if ( null !== $args['render_callback'] ) {
+	if ( is_callable( $args['render_callback'] ) ) {
 		$block_render_callback   = $args['render_callback'];
 		$args['render_callback'] = function( $attributes, $content, $block ) use ( $block_render_callback ) {
```

Comment:
I observed an error `ArgumentCountError : Too few arguments to function {closure}(), 2 passed in …` in core tests with latest Gutenberg. It calls `WP_Block_Type::render()`. I believe in order to prevent that we'd need to make the third argument optional like this:

```suggestion
		$args['render_callback'] = function( $attributes, $content, $block = null ) use ( $block_render_callback ) {
```

Reply:
Agreed with making the third param optional. For some reason I thought we were already doing this. :/

Reply:
Nice catch, ty!